spec/controllers/api/v1/items_controller_spec.rb

```diff
@@ -0,0 +1,13 @@
+RSpec.describe Api::V1::ItemsController do
```

Comment:
Let's pair today and work through deciding exactly how we want to make the changes going forward together.